public/pages/CreateTrigger/containers/ConfigureActions/ConfigureActions.js

```diff
@@ -187,6 +188,12 @@ class ConfigureActions extends React.Component {
       let error = null;
       if (response.ok) {
         error = checkForError(response, error);
+        if (!_.isEmpty(action.destination_id)) {
```

Comment:
Would it make more sense to always use destination name as this `sendTo` field?

Reply:
I was debating that, but my thoughts were preferring to display the action name so that the toast is a little more personalized. In addition, if the action name is present, the function can skip over iterating through the list of destinations for the destination name. I have no strong preferences either way.